addons/microsoft_outlook/models/microsoft_outlook_mixin.py

```diff
@@ -81,7 +81,7 @@ def open_microsoft_outlook_uri(self):
         if not self.use_microsoft_outlook_service or not self.is_microsoft_outlook_configured:
             raise UserError(_('Please configure your Outlook credentials.'))
 
-        if not self.smtp_user:
+        if not self.user:
```

Comment:
```suggestion
        if "smtp_user" in self._fields and not self.smtp_user:
```
Same logic as before